components/frontend/src/dashboard/StatusPieChart.js

```diff
@@ -12,10 +12,10 @@ nrMetricsLabel.PropTypes = {
     nrMetrics: number,
 }
 
-export function StatusPieChart({ animate, colors, label, tooltip, summary, style, maxY, width, height }) {
+export function StatusPieChart({ animate, colors, events, height, label, maxY, style, summary, tooltip, width }) {
     const nrMetrics = sum(summary)
     const outerRadius = 0.45 * Math.min(height, width)
-    const minInnerRadius = 0.3 * outerRadius
+    const minInnerRadius = 0.4 * outerRadius
```

Comment:
Where do these magic numbers come from?

Reply:
Trial and visual inspection I'm afraid. There needs to be enough room in the center for the label (nr of metrics) and that depends on font size.